src/PIL/Image.py

```diff
@@ -1272,6 +1272,98 @@ def getdata(self, band=None):
             return self.im.getband(band)
         return self.im  # could be abused
 
+    def getdominantcolors(self, numcolors=3, maxiter=50, threshold=1):
+        """
+        Returns a list of dominant colors in an image using k-means
+        clustering.
+
+        :param numcolors: Number of dominant colors to search for.
+        The default number is 3.
+        :param maxiter: Maximum number of iterations to run the
+        algorithm.  The default limit is 50.
+        :param threshold: Early stopping condition for the algorithm.
+        Higher values correspond with increased color differences. The
+        default is set to 1 (corresponding to 1 pixel difference).
+        :returns: An unsorted list of (pixel) values.
```

Comment:
```suggestion
        :returns: An unsorted list of pixel values.
```
I suspect you copied this from the `getcolors` docstring. The brackets there were part of indicating the tuple contents. This function doesn't necessarily return a list of tuples.